docs/src/modules/utils/postProcessImport.ts

```diff
@@ -1,11 +1,11 @@
 export const DATE_ADAPTER_VERSIONS: Record<string, string> = {
   'date-fns': '^2.30.0',
   'date-fns-jalali': '^2.30.0-0',
-  dayjs: '^1.11.10',
-  luxon: '^3.4.4',
-  moment: '^2.29.4',
-  'moment-hijri': '^2.1.2',
-  'moment-jalaali': '^0.10.0',
+  dayjs: '^1.11.13',
+  luxon: '^3.5.0',
+  moment: '^2.30.1',
+  'moment-hijri': '^3.0.0',
+  'moment-jalaali': '^0.10.1',
```

Comment:
Can't we fetch these from package.json instead of manually have to change?

Reply:
Great idea, I'll look into it. 👍